SleepNumberController_App.groovy

```diff
@@ -304,18 +305,18 @@ def scheduledRefreshChildDevices() {
   }
 }
 
-def refreshChildDevices() {
+void refreshChildDevices() {
   // Only refresh if mode is a selected one
   if (settings.modes && !settings.modes.contains(location.mode)) {
     return
   }
   getBedData()
   updateLabel()
 }
-
-def refreshChildDevices(ignored, ignoredDevId) {
+/*
+void refreshChildDevices(ignored, ignoredDevId) {
```

Comment:
This is required so the driver can call this method (it sends two arguments so the no argument variant won't work).

Reply:
ok, you can reject that block